configure.acr

```diff
@@ -75,6 +75,11 @@ CHKFNC c explicit_bzero
 CHKFNC c explicit_memset
 CHKFNC c clock_nanosleep
 CHKFNC c sigaction
+CHKFNC c clock_gettime
+
+IFNOT HAVE_CLOCK_GETTIME {
```

Comment:
this is wrong. you cant add ldflags in configure, this must be done in a makefile. and consider that this thing is a linuxism, no other unix system have this stupid librt